### ISSUE TITLE
How about yolov4-p6-light in 640 size?

### ISSUE THREAD

**Issue by Zigars** [WongKinYiu/yolor#5]
I have test trained yolor-p6.yaml In VisDrone dataset (a famous UAV dataset) use 640 size and your pretrained .pt. And I get a excellent results, It's a great work!
Then, I trained yolov4-p6-light.yaml(remove the reOrg and IDetect module and I fix it in **yolov5's rep**) in VisDrone dataset use 640 size without pretrained, but the results under the original yolov4-csp‘s result. maybe it have some bug in my code.
So I'm training the same yolov4-p6-light.yaml(remove the reOrg and IDetect module and I fix it in your **yolor's rep**) in VisDrone use 640 size without pretrained, and I need find the bug in my own code if your yolor rep can get a good results. If not, that says yolov4-p6-light need a coco pretrained? because the model have four output, the loss can hard to convergence?
The all experiment trained in 300epoch, 32batchsize, 640*640 size, use single V100 to train the model. maybe you can solve my question. thks!

**Comment by Zigars** [comment]
my test is over, and these are my test results, I don't know which part have different, my VisDrone-yolov4 Rep is forked by yolov5's latest version, maybe the calculate of mAP have different? 
![Train](https://user-images.githubusercontent.com/32835472/118747357-70109800-b88c-11eb-9314-c39568e11ee8.png)
![Test](https://user-images.githubusercontent.com/32835472/118747468-9e8e7300-b88c-11eb-8bcc-7104e4664198.png)

**Comment by WongKinYiu** [comment]
Calculation of Precision and Recall are different, yolov5 calculate average score of 0.5:0.95 and yolor calculate score of 0.5.
And I think AP(0.5) and AP(0.5:0.95) are using same calculation.

**Comment by Zigars** [comment]
If that your explain is true, I think yolov4-p6-light.yaml (without reOrg and IDetect module) can not catch the yolov4-csp.yaml results in VisDrone dataset,  although yolov4-p6-light.yaml have four output, the same infer time as yolov4-csp.yaml. I'm confuse about it. thank you for your explain!
these are my yolov4-p6-light.yaml and yolov4-csp.yaml cfg files.yolov4-csp is a great work, I'm trying to make it more applicable for the VisDrone dataset.
[VisDrone-yolov4-cfg.zip](https://github.com/WongKinYiu/yolor/files/6506194/VisDrone-yolov4-cfg.zip)

**Comment by WongKinYiu** [comment]
In my experiments.
input resolution: performance
1280: p6 > p7 > p5
1536: p7 > p6 > p5

maybe for 640 case, p5 models will get better performance than p6 models, but i have not tested it yet.

**Comment by Wanghe1997** [comment]
Zigars，我可以加你的QQ或者微信吗？我是一个学生，我想用自己的数据集跑作者的YOLOR，但是没跑通，可以请教下您吗？

**Comment by TheConstant3** [comment]
> In my experiments.
> input resolution: performance
> 1280: p6 > p7 > p5
> 1536: p7 > p6 > p5
> 
> maybe for 640 case, p5 models will get better performance than p6 models, but i have not tested it yet.

@WongKinYiu 
My experiments on custom dataset with 30 epochs, batch size 4 on pretrained yolor-p6, showed that input size 640 much better then 1280
I have images with different resolutions in dataset
And use your hyp files for 640 and 1280 respectively
Do I need to change batch size or epochs with different input size for more correct results?

Thanks in advance!!
